misc/releaseprep

```diff
@@ -86,6 +86,8 @@ fix_patch_h() {
 		cat src/patch.h | sed 's/^patch.*Git.*Git version \*\//patch("PRE-RELEASE");           \/* RC version *\//g' > src/patch.h_
 	else
 		cat src/patch.h | sed 's/^patch.*/\/* PATCH GOES HERE *\//g' > src/patch.h_
+		mv src/patch.h_ src/patch.h
+		cat src/patch.h | sed -e '0,/^\/\* PATCH GOES HERE \*\//s//patch("");/' > src/patch.h_
```

Comment:
Holy cannoli, please kill the cats. I also don't know the machine where the releases are prepared on, but I'd be surprised if the sed-binary on it wouldn't have the -i functionality. To top it off, those two sed-invocations can be combined with a mere ';'.
This results in the above 3 lines becoming:
`sed -i 's/^patch.*/\/* PATCH GOES HERE *\//g;0,/^\/\* PATCH GOES HERE \*\//s//patch("");/' src/patch.h`
and obsoleting the line below (under condition that the "cat | sed > *_" scheme is changed in the first if-clause).

The above comments apply for this whole script.